database/migrations/2023_04_24_075312_add_owner_id_to_repositories_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('repositories', static function (Blueprint $table) {
+            // third party user_id (ex: github user id)
+            $table->string('owner_id');
```

Comment:
instead of commenting here, maybe rename the column by owner_github_id ? (its a key on user table)

Reply:
owner_third_party_id is correct 🤷‍♂️